app/src/Homepage/Homepage-components.tsx

```diff
@@ -20,12 +42,21 @@ export const Sidebar: React.FC<SidebarProps> = ( props: SidebarProps ) => {
     );
 }
 
+export const GetClassNumber = ( label : string ) => {
+    return label.split(" ", 2)[1];
```

Comment:
I forgot to add this but can you make sure that GetClassNumber will return a valid number? That way it doesnt return nothing/not a number. Then can you add another unit test that ensure it throws an exception (a descriptive exception like InvalidOperationException if that exists...)

Reply:
Right now the function actually just retrieves the second word from a string as a string, including the ‘s’ in ‘X00s’ because that’s included in the URL, so the logic would have to change. If the function will be used for a purpose other than these URLs, it’d make sense to change, but I don’t know if it will yet. Want me to change it to a number parsing function?

Reply:
No, its fine to leave as is then